packages/scripts/README.md

```diff
@@ -117,6 +117,20 @@ _Flags_:
 - `--gpl2`: Validates against [GPLv2 license compatibility](https://www.gnu.org/licenses/license-list.en.html)
 - `--ignore=a,b,c`: A comma-separated set of package names to ignore for validation. This is intended to be used primarily in cases where a dependency’s `license` field is malformed. It’s assumed that any `ignored` package argument would be manually vetted for compatibility by the project owner.
 
+### `create-block`
+
+Takes a new name for a block and creates a minimal plugin development environment for it in the current working directory.
+
+_Example:_
+
+```sh
+$ wp-scripts create-block "My Fancy Block"
+$ cd ./my-fancy-block-plugin
```

Comment:
If the plugin is called "My Fancy Block" does it append `-plugin` to the folder created?

If not, then: 

```suggestion
$ cd ./my-fancy-block
```

Reply:
>If the plugin is called "My Fancy Block" does it append -plugin to the folder created?

Yes, "My Fancy Block" is the block name.

Reply:
Why append `-plugin` to the plugin name? Why not just `my-fancy-block` for the slug?

Reply:
So that they are not the same.

Is there a convention for this?

Reply:
With `create-react-app` whatever name I use is converted to snake-case, no extra `-app` or `-plugin` is appended to the name.

And for the most part if I would expect the name to be what name I gave, if I gave "My Fancy Block Plugin" then I _would_ expect the `-plugin` included in the slug

Reply:
> So that they are not the same.

Is there any concerns where having the plugin name and plugin slug the same that would be of concern?

Reply:
>With create-react-app whatever name I use is converted to snake-case, no extra -app or -plugin is appended to the name.

Yes, because you are naming the app, and the app only.

>And for the most part if I would expect the name to be what name I gave, if I gave "My Fancy Block Plugin" then I would expect the -plugin included in the slug

You are not naming the plugin, you are naming the block, and the plugin name gets generated from that.

>Is there any concerns where having the plugin name and plugin slug the same that would be of concern?

No, I think it's just a style thing. I don't have a strong opinion on it.